### PR TITLE
feat: fill days of prev and next month in calendar

### PR DESCRIPTION
**Pull request by d-Rickyy-b**
Instead of only using empty spaces, it would be cool to actually show days of the previous/next month. I implemented a method getPrefill & getPostfill and used them in the njk template. Also both prefill & postfill got their own config value (which defaults to true). They return either empty lists if the setting was set to false or return the days of the prev/next month if the setting was set to true.

### CODE REVIEW

File: MMM-MonthCalendar.js [hunk 1]
```diff
@@ -44,22 +46,55 @@ Module.register("MMM-MonthCalendar", {
...
+        return days;
+    },
+
+    getCalendarDays: function() {
+        let days = [];
+
         for (let day = 1; day <= moment().daysInMonth(); day++) {
             days.push(day);
         }
-        Log.info(days);
```
Comment (PalatinCoder) [inline MMM-MonthCalendar.js:61]: Good catch. I forgot this one 😄

**Review by PalatinCoder — CHANGES_REQUESTED** [review]
Implementation is pretty much how I would've done it. However, I think the `preFill` and `postFill` config values should be combined into one. What's the point of showing the next month's days but not the previous ones? 😉

**Comment by d-Rickyy-b** [comment]
> 
> 
> Implementation is pretty much how I would've done it. However, I think the `preFill` and `postFill` config values should be combined into one. What's the point of showing the next month's days but not the previous ones? 😉

I was pretty convinced at first to have only one, but then I thought about splitting it for more control. But you are right, why would users want to show any (prev/next) month but not the other 😆 - lemme change that real quick.

**Review by PalatinCoder — APPROVED** [review]
LGBR 👍

**Pull request merged**